docs/CUSTOM_RULES_101.md

```diff
@@ -346,11 +347,21 @@ Test passed? If not, then check if you somehow missed a step.
 
 If it passed...
 
-> ***
+>
 > :tada: **Congratulations!** :confetti_ball:
 >
 > [*You implemented your first custom rule for the SonarQube Java Analyzer!*](resources/success.jpg)
-> ***
+>
+
+### What you can use, and what you can't
+
+As a rule of thumb, you should keep in mind when writing custom rules to **always**, and **only**, use classes from the [`org.sonar.plugins.java.api`](https://github.com/SonarSource/sonar-java/tree/5.12.1.17771/java-frontend/src/main/java/org/sonar/plugins/java/api) packages from the SonarJava plugin. 
```

Comment:
Remove "As a rule of thumb, you should keep in mind", [rule of thumb](https://en.wikipedia.org/wiki/Rule_of_thumb) means that it is not a precise rule, only some guidance, which is not the case here, as this is a very strict requirement. 
Suggestion: "When writing custom rules your plugin can only use classes from package `org.sonar.plugin.api` "